src/Robo/Commands/Artifact/AcHooksCommand.php

```diff
@@ -60,29 +59,17 @@ public function postCodeDeploy($site, $target_env, $source_branch, $deployed_tag
    * @throws \Exception
    */
   public function postCodeUpdate($site, $target_env, $source_branch, $deployed_tag, $repo_url, $repo_type) {
-    $this->dieIfAcsfEnv($target_env);
```

Comment:
This is here to intentionally prevent a post-deploy hook from being executed an ACSF, given that it is a dangerous practice that is discouraged.

@lcatlett right?

Reply:
@grasmash the issue is that with the exception it kills the deployment. the intent with this PR is to still "not" do the cloud hook, but not throw an exception in the process (so fail gracefully)

Reply:
Failing gracefully on post-code-update looks fine, however I recommend adding the exception to kill the deployment to postCodeDeploy since that is the most problematic of the Cloud Hooks - it may be up for debate whether to throw the exception and terminate the deployment if $env is 01live / production as well as 01update, as using post-code-deploy on an ACSF code update in production is dangerous in all usages since it often will bypass the ACSF update environment during deployments.

Reply:
Just to be clear, I doubt we want to throw an _exception_ on post-code-deploy. That would make it impossible to maintain a codebase on ACE and ACSF simultaneously. Rather, we want to bail gracefully on post-code-deploy on ACSF.